pdf/pdfcanvas.h

```diff
@@ -29,15 +29,24 @@ class PDFDocument;
 class PDFCanvas : public QQuickItem
 {
     Q_OBJECT
+    Q_ENUMS(PageRotation)
     Q_PROPERTY(PDFDocument* document READ document WRITE setDocument NOTIFY documentChanged)
     Q_PROPERTY(QQuickItem* flickable READ flickable WRITE setFlickable NOTIFY flickableChanged)
     Q_PROPERTY(float spacing READ spacing WRITE setSpacing NOTIFY spacingChanged)
     Q_PROPERTY(QColor linkColor READ linkColor WRITE setLinkColor NOTIFY linkColorChanged)
     Q_PROPERTY(QColor pagePlaceholderColor READ pagePlaceholderColor WRITE setPagePlaceholderColor NOTIFY pagePlaceholderColorChanged)
     Q_PROPERTY(int currentPage READ currentPage NOTIFY currentPageChanged)
     Q_PROPERTY(float linkWiggle READ linkWiggle WRITE setLinkWiggle NOTIFY linkWiggleChanged)
+    Q_PROPERTY(PageRotation pageRotation READ pageRotation WRITE setPageRotation NOTIFY pageRotationChanged)
 
 public:
+    enum PageRotation {
+        NO_ROTATION,
+        CLOCKWISE,
+        ANTI_CLOCKWISE,
+        UP_SIDE_DOWN
```

Comment:
We don't commonly have all-caps enums, but rather CamelCased ones. Also something like counterclockwise or inverted for naming?

Reply:
That's right, I'm going to change and CounterClockWise seems more Englisk indeed.

Reply:
Case changed.